SpectraLogic.EscapePodClient.Integration.Test/EscapePodClientFixture.cs

```diff
@@ -41,5 +53,32 @@ static EscapePodClientFixture()
 
             EscapePodClient = escapePodClientBuilder.Build();
         }
+
+        private static void CreateDevice()
+        {
+            DeviceName = ConfigurationManager.AppSettings["DeviceName"];
+            //TODO implement this after ESCP-131 is done
+        }
+
+        private static void CreateArchive()
+        {
+            ArchiveName = ConfigurationManager.AppSettings["ArchiveName"];
+            var getArchiveRequest = new GetArchiveRequest(ArchiveName);
+            try
+            {
+                var archive = EscapePodClient.GetArchive(getArchiveRequest);
```

Comment:
I'll add a head call for this so that you can add an exists call that doesn't require you to get the response body.

Reply:
:+1:  i'll update once available